R/surv-brier_survival_integrated.R

```diff
@@ -91,6 +91,14 @@ brier_survival_integrated_vec <- function(truth,
     truth, estimate, case_weights
   )
 
+  num_eval_times <- get_unique_eval_times(estimate)
+  if (num_eval_times < 2) {
+    cli::cli_abort(
+      "At least 2 evaluation time{?s} {?is/are} required. \\
+      Only {num_eval_times} unique time{?s} {?was/were} given."
```

Comment:
I think the pluralization is overkill here :) The requirement is always "at least 2" and `{num_eval_times}` is not going to be 2 or more when we trigger that error.

Reply:
new and improved in https://github.com/tidymodels/yardstick/pull/470